### PR TITLE
Update recipe to version 2.6.2

### PR DESCRIPTION
**Pull request by jjhelmus**
Closes https://github.com/conda-forge/flex-feedstock/pull/4
Closes https://github.com/conda-forge/flex-feedstock/pull/5

Incorporates changes recommended in #4 and #5

Edit by @jakirkham : Added closes notes about to close the other PRs this supersedes.

### CODE REVIEW

**Comment by jjhelmus** [comment]
I believe a conda package of `help2man` could be made installing it with yum is undesired.  The latest source can be downloaded at https://ftp.gnu.org/gnu/help2man/.  Looks to be a perl script.

**Comment by jjhelmus** [comment]
Looks like a `help2man` package will be needed as this is not installed on OS X.

**Comment by jjhelmus** [comment]
Added a recipe for `help2man` to staged-recipes, conda-forge/staged-recipes#2049.  Will update this PR once this is merged.

**Comment by jakirkham** [comment]
I think we can get around that req or at least it seems others are able too.

xref: http://www.linuxfromscratch.org/lfs/view/systemd/chapter06/flex.html

**Comment by jjhelmus** [comment]
> I think we can get around that req or at least it seems others are able too.

And the tarball has manpages included... odd.

**Comment by jakirkham** [comment]
Am ok merging this now or waiting for `help2man`. What would you like to do?

**Comment by jjhelmus** [comment]
> What would you like to do?

I'd rather use the `help2man` package from `conda-forge` now that it is available.  I've  updated the recipe to reflect this.

**Comment by jjhelmus** [comment]
This should be ready to go.  Are any changes needed before merging?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,34 +1,24 @@
...
 
 build:
   skip: true  # [win]
-  number: 1
+  number: 0
   detect_binary_files_with_prefix: true
 
 requirements:
   build:
-    - autoconf
```
Comment (ocefpaf) [inline recipe/meta.yaml:26]: The change in the tarball source is OK but `autoconf` should work now.

**Comment by jakirkham** [comment]
Thanks @jjhelmus .

**Pull request merged**